sql/core/src/test/resources/sql-tests/inputs/datetime.sql

```diff
@@ -2,13 +2,16 @@
 
 -- [SPARK-31710] TIMESTAMP_SECONDS, TIMESTAMP_MILLISECONDS and TIMESTAMP_MICROSECONDS to timestamp transfer
 select TIMESTAMP_SECONDS(1230219000),TIMESTAMP_SECONDS(-1230219000),TIMESTAMP_SECONDS(null);
+select TIMESTAMP_SECONDS(1.23), TIMESTAMP_SECONDS(1.23d);
```

Comment:
Since this has `Decimal` and `Double`, can we have `Float` together by using `TIMESTAMP_SECONDS(CAST(1.23 AS FLOAT))`?